ZEP/instructions/zep0000.md

```diff
@@ -0,0 +1,201 @@
+necessarily represent Zarr community consensus or recommendation, so users and implementers are free to ignore informational ZEPS or follow their advice.
+
+> Process ZEP
+
+Describes a new process around Zarr and its implementations, or proposes a change to(or an event in) a process. Process ZEPs are like Standard Track ZEPs but apply to areas other than the Zarr project itself. 
+They may propose an implementation, but not to Zarr’s codebase; they require community consensus; unlike informational ZEPs, they are more than recommendations, and users are typically not free to ignore them. 
+Examples include procedures, guidelines, changes to the decision-making process, and changes to the tools or environment used in Zarr’s development. Any meta-ZEP is also considered a Process ZEP.
+
+## ZEP Workflow
+
+The ZEP process begins with a new idea for Zarr. It is highly recommended that a single ZEP contain a single key proposal or new idea. Small enhancements or 
+patches often don’t need a ZEP and can be injected into the Zarr development workflow with a pull request to the Zarr ZEP repo. The more focused the ZEP, 
+the more successful it tends to be. If in doubt, split your ZEP into several well-focused ones.
+
+Each ZEP must have a champion—someone who writes the ZEP using the style and format described below, shepherds the discussions in the appropriate forums, 
+and attempts to build community consensus around the idea. The ZEP champion (a.k.a. Author) should first attempt to ascertain whether the idea is suitable for a ZEP. 
+Posting to the [Gitter](https://gitter.im/zarr-developers/community) is the best way to go about doing this.
+
+Vetting an idea publicly before going as far as writing a ZEP is meant to save the potential author time. Asking the Zarr community first if an idea is original 
+helps prevent too much time being spent on something that is guaranteed to be rejected based on prior discussion. It also helps to make sure the idea applies to 
+the entire community and not just the author. Just because an idea sounds good to the author does not mean it will work for most people.
+
+Once the champion has asked the Zarr community whether an idea has any chance of acceptance, a draft ZEP should be presented to the appropriate venue mentioned 
+above. This gives the author a chance to flesh out the draft ZEP to make properly formatted, of high quality, and to address initial concerns about the proposal.
+
+After the PR for the ZEP is in place, a post should be made on the Gitter channel containing the sections up to “Backward compatibility”, to limit discussion 
+there to usage and impact. Discussion on the pull request will have a broader scope, also including details of implementation.
+
+Standards Track ZEPs consist of two parts, a design document and a reference implementation. It is generally recommended that at least a prototype implementation 
```

Comment:
If `Standards Track` means something other than "impacts all implementations", then I could see adding that as an additional category.

Reply:
>  If `Standards Track` means something other than "impacts all implementations",

`Standards Track ZEP` are something that introduces a new feature to the Zarr. It's up to the author to mention that their ZEP affects a single implementation or all of the implementations. Whatever the case maybe they need to add prototype implementation for what they've mentioned in their ZEP.

> then I could see adding that as an additional category.

Do you mean having `Standard Track ZEP` for every implementation? Like `Standard Track ZEP - Python`, `Standard Track ZEP - Java` etc.?